docs/how/how_custom_visualisation.rst

```diff
@@ -79,6 +79,8 @@ There are several constant parameters available, each detailed below with the va
 		int SelectedSample;
 		uvec4 YUVDownsampleRate;
 		uvec4 YUVAChannels;
+		float RangeMin;
+		float RangeMax;
```

Comment:
This is good, but I also meant to add a section like the ones below (look at "Selected Mip level" describing `RENDERDOC_SelectedMip`) explaining exactly what RangeMin/RangeMax mean :grinning: .